src/app/services/challenge.service.ts

```diff
@@ -58,6 +58,15 @@ export class ChallengeService {
     this.challengeHostSource.next(isChallengeHost);
   }
 
+   /**
+   * get if the value to be submitted is valid
```

Comment:
Can we please change the comment to `Returns boolean: if the max submissions count is valid` or something else which makes more sense